src/app/app.component.ts

```diff
@@ -76,7 +76,7 @@ export class HeaderComponent<D> implements OnInit {
    *
    * @param {"Gregorian" | "Julian"} calendar the target calendar format.
    */
-  convertCalendar(calendar: 'Gregorian' | 'Julian') {
+  convertCalendar(calendar: 'Gregorian' | 'Julian' | 'Islamic') {
 
     if (this._dateAdapter instanceof JDNConvertibleCalendarDateAdapter) {
 
```

Comment:
@mmalerba After conversion to the Islamic calendar, the weekdays are not updated:

<img width="328" alt="conversion" src="https://user-images.githubusercontent.com/6000023/61772428-89606e80-adf2-11e9-818d-aa08eb05a046.png">

However, when I close and reopen the datepicker widget, the names are updated correctly. 

Do I have to call an additional method after each conversion (similar to `updateTodaysDate`)?